src/rtextures.c

```diff
@@ -2118,6 +2118,65 @@ void ImageFlipHorizontal(Image *image)
     }
 }
 
+// Rotate image in degrees
+void ImageRotate(Image *image, int degrees)
+{
+    // Security check to avoid program crash
+    if ((image->data == NULL) || (image->width == 0) || (image->height == 0)) return;
+
+    if (image->mipmaps > 1) TRACELOG(LOG_WARNING, "Image manipulation only applied to base mipmap level");
+    if (image->format >= PIXELFORMAT_COMPRESSED_DXT1_RGB) TRACELOG(LOG_WARNING, "Image manipulation not supported for compressed formats");
+    else
+    {
+        double rad = degrees * PI / 180.0;
+        double sin_rad = sin(rad);
```

Comment:
Please could you use `float` instead of `double`? I tried to avoid `double` usage in raylib.

Please, could you also review code to follow [raylib coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md)?

Reply:
Resolved as requested